packages/inference/src/types.ts

```diff
@@ -87,6 +87,7 @@ export type RequestArgs = BaseArgs &
 		| { text: string }
 		| { audio_url: string }
 		| ChatCompletionInput
+		| FeatureExtractionInput
```

Comment:
Not needed AFAIK (already covered by `{ inputs: unknown }` at L85)

Reply:
handled in https://github.com/huggingface/huggingface.js/pull/1239/commits/c57de75cdc6d6298085c8e6c3a4fa7a5e6dacceb